src/settings.svelte

```diff
@@ -215,6 +224,12 @@
                 >{$_(`settings.${item}.title`)}</div>
             {/each}
         </div>
+        
+        
+        <div class="save-button" >
+            <!-- svelte-ignore missing-declaration -->
+            <button class="button" on:click={()=>Neutralino.events.dispatch("windowClose")} >✔️</button>
+        </div>
```

Comment:
Please provide screenshots when adding new UI elements to so we can determine if they fit in or not